tests/IntegrationTests/WorkflowExecutor.IntegrationTests/Features/PayloadCollection.feature

```diff
@@ -19,30 +19,24 @@ Tests around the payload collection
 @PatientDetails
 Scenario: Seed MinIO with patient metadata, and payload collection has details
```

Comment:
Does scenario name need updating to reflect what's happening now?

Reply:
Agreed, renamed